### ISSUE TITLE
Cannot install Flood

### ISSUE THREAD

**Issue by garikaib** [liaralabs/swizzin#236]
**Describe the bug**
I am failing to install Flood on Ubuntu 18.04.3

**To Reproduce**
Steps to reproduce the behaviour:
1. Install package 'flood' using command 'box install flood'
2. Go to 'example.com/flood'
3. Error '502' Gateway error. Flood does not even appear in panel

**Expected behavior**
Flood is installed
**Server Info (please complete the following information):**
 - OS: Ubuntu 18.04.3

**Additional context**
The install script for flood has this on line 20:
`apt-get -y -q install nodejs build-essential npm`
This gives an error:
The following packages have unmet dependencies:
 nodejs : Conflicts: npm
I haven't had time to fully investigate this but this is usually a PPA issue. Multiple versions of packages conflicting with some having unment dependencies.

**Comment by liaralabs** [comment]
Ok this should be properly fixed at the moment. Tested across 9/10 16.04/18.04.

npm should absolutely be getting installed in the `nodejs` packages if your repositories were properly setup by the nodesource command. So, I've removed the command to explicitly install it *because* it will cause conflicts if you try to install it against your distro's version of npm.

If `npm` isn't installed after my setup commands, there's likely a repository issue somewhere and I'd be willing to be the node source repo hasn't been installed.
